src/Components/userDashboard/ResearchStatement.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import UpdateIcon from '@material-ui/icons/Update';
+
+const useStyles = makeStyles(() => ({
+    root: {
+        minHeight: '100vh',
+        flex: '1',
+    },
+    title: {
+        marginTop: '50px',
+        textAlign: 'center',
+        padding: '10px'
+    },
+    btnContainer: {
+        display: 'flex',
+        width: '65%',
+        justifyContent: 'space-between',
+        margin: '20px auto'
+    },
+    editBtnContainer: {
+        display: 'flex',
+        justifyContent: 'space-between',
+        width: '200px',
+        marginLeft: 'auto' 
+    },
+    txt:{
+        fontSize: '20px',
+        margin: '20px auto',
+        width: '65%',
+        minHeight: '90vh',
+        fontFamily: 'sans-serif',
+        backgroundColor: 'white',
+        padding: '10px',
+        borderRadius: '10px',
+        boxShadow: '0 0 10px rgba(0,0,0,0.1)'
+    },
+    txtArea: {
+        display: 'block',
+        margin: '20px auto',
+        width: '65%',
+        minHeight: '90vh',
+        fontSize: '20px',
+        padding: '10px',
+    },
+    '@media screen and (max-width: 992px)': {
+        btnContainer: {
+            width: '90%'
+        },
+        txt:{
+            width: '90%'
+        },
+        txtArea: {
+            width: '90%'
+        }
+    }
```

Comment:
Use the theme breakpoints feature given by Material UI, like here https://material-ui.com/customization/breakpoints/